include/swift/AST/AutoDiff.h

```diff
@@ -465,6 +466,10 @@ struct SILAutoDiffIndices {
 
   bool operator==(const SILAutoDiffIndices &other) const;
 
+  bool operator!=(const SILAutoDiffIndices &other) const {
```

Comment:
I don't see any use in this file.

Reply:
I used `!=` on occasion while debugging this and other patches. I'd like to keep it, if that's fine.